react/react-training/src/screens/MyScreenComponent/Board.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import Square from './Square';
+
+class Board extends Component {
+  renderSquare(i) {
+    return <Square value={this.props.squares[i]} onClick={() => this.props.onClick(i)} />;
+  }
+
+  render() {
+    return (
+      <div>
```

Comment:
Change this for a `Fragment`

Reply:
This change is made, but I think that it's better in another way. Currently, with your recommendation, we have:

``` 
// Game:
    return (
      <div className={styles.game}>
        <div className={styles.gameBoard}>
          <Board squares={current.squares} onClick={i => this.handleClick(i)} />
        </div>

// Board: 
  render() {
    return (
      <fragment>
        <div className={styles.status}>{status}</div>
           ...
```

I think it's better the board has the div and can be work independently:

``` 
// Game:
    return (
      <div className={styles.game}>
          <Board squares={current.squares} onClick={i => this.handleClick(i)} />

// Board: 
  render() {
    return (
      <div className={styles.gameBoard}>
        <div className={styles.status}>{status}</div>
           ...
```

What do you think?